fmriprep/workflows/bold/resampling.py

```diff
@@ -712,11 +640,175 @@ def init_bold_preproc_report_wf(mem_gb, reportlets_dir, name='bold_preproc_repor
     return workflow
 
 
-def _select_template(template, template_specs):
+def init_bold_grayords_wf(
+    grayord_density,
+    mem_gb,
+    repetition_time,
+    name='bold_grayords_wf'
+):
+    """
+    Sample Grayordinates files onto the fsLR atlas.
+
+    Outputs are in CIFTI2 format.
+
+    Workflow Graph
+        .. workflow::
+            :graph2use: colored
+            :simple_form: yes
+
+            from fmriprep.workflows.bold import init_bold_grayords_wf
+            wf = init_bold_grayords_wf(mem_gb=0.1, grayord_density='91k')
+
+    Parameters
+    ----------
+    grayord_density : :obj:`str`
+        Either `91k` or `170k`, representing the total of vertices or *grayordinates*.
+    mem_gb : :obj:`float`
+        Size of BOLD file in GB
+    name : :obj:`str`
+        Unique name for the subworkflow (default: ``'bold_grayords_wf'``)
+
+    Inputs
+    ------
+    bold_std : :obj:`str`
+        List of BOLD conversions to standard spaces.
+    spatial_reference :obj:`str`
+        List of unique identifiers corresponding to the BOLD standard-conversions.
+    subjects_dir : :obj:`str`
+        FreeSurfer's subjects directory.
+    surf_files : :obj:`str`
+        List of BOLD files resampled on the fsaverage (ico7) surfaces.
+    surf_refs :
+        List of unique identifiers corresponding to the BOLD surface-conversions.
+
+    Outputs
+    -------
+    cifti_bold : :obj:`str`
+        List of BOLD grayordinates files - (L)eft and (R)ight.
+    cifti_variant : :obj:`str`
+        Only ``'HCP Grayordinates'`` is currently supported.
+    cifti_metadata : :obj:`str`
+        Path of metadata files corresponding to ``cifti_bold``.
+    cifti_density : :obj:`str`
+        Density (i.e., either `91k` or `170k`) of ``cifti_bold``.
+
+    """
+    import templateflow.api as tf
+    from niworkflows.interfaces.cifti import GenerateCifti
+    workflow = Workflow(name=name)
+    workflow.__desc__ = """\
+*Grayordinates* files [@hcppipelines] containing {density} samples were also
+generated using the highest-resolution ``fsaverage`` as intermediate standardized
+surface space.
+""".format(density=grayord_density)
+
+    fslr_density = ('59k', '32k')[grayord_density == '91k']
```

Comment:
for consistency with below:
```suggestion
    fslr_density, mni_density = ('32k', '2') if grayord_density == '91k' else ('59k', '1')
```

Reply:
this looks much cleaner